Tests/Fuzzers/FuzzTarget.cpp

```diff
@@ -1,87 +1,76 @@
-#include <iostream>
-
-#include <IPv4Layer.h>
-#include <Packet.h>
 #include <PcapFileDevice.h>
+#include <Packet.h>
+#include <Logger.h>
+#include "DumpToFile.h"
+#include "ReadParsedPacket.h"
 
-#include "Logger.h"
-
-#define TMP_FILEPATH "/tmp/fuzz_sample.pcap"
+static std::string tmpName;
+static std::string tmpFile;
 
-// This function is created as PcapPlusPlus doesn't seem to offer a way of
-// parsing Pcap files directly from memory
-int dumpDataToPcapFile(const uint8_t *data, size_t size)
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
-	FILE *fd;
-	int written = 0;
+	if (tmpName.empty())
+		tmpName = tmpnam (NULL);
 
-	fd = fopen(TMP_FILEPATH, "wb");
-	if (fd == NULL)
-	{
-		std::cerr << "Error opening pcap file for writing\n";
-		return -1;
-	}
+	if (tmpFile.empty())
+		tmpFile = tmpName + FILE_EXT;
```

Comment:
Why are these lines needed? 🤔

Reply:
The file name was hardcoded when we had only one fuzzer. This change allows each fuzzer to have a separate file name. However for performance reasons it is initialized only once. `tmpFile` is static and empty in the beginning. `LLVMFuzzerTestOneInput` is called millions of times in the same process.